flowkit/config/dependency.go

```diff
@@ -0,0 +1,79 @@
+/*
+ * Flow CLI
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package config
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/onflow/flow-go-sdk"
+)
+
+func ParseRemoteSourceString(s string) (network, address, contractName string, err error) {
+	parts := strings.Split(s, "://")
+	if len(parts) != 2 {
+		return "", "", "", fmt.Errorf("invalid dependency source format")
```

Comment:
When using `Errorf` it's best to add more context, like the `s` string that caused the error:
```go
fmt.Errorf("invalid dependency source format: %s", s)
```
If you don't have any args you can use `errors.New(str)` (https://pkg.go.dev/errors#New)

Reply:
Updated